### ISSUE TITLE
This error always occurs after compilation and cannot be used normally.

### ISSUE THREAD

**Issue by xiawiner** [ExistedGit/Dota2Cheat#202]
Dear author, I always get errors after compiling with vs2022. I compile exactly according to your tutorial. Why?

![捕获](https://github.com/ExistedGit/Dota2Cheat/assets/20815754/5b502243-6cf3-4082-9a86-ac1de9377887)
I really hope to get a reply, thank you

**Comment by ExistedGit** [comment]
Do you get compile errors or are you concerned about warnings at runtime?

Warnings like the ones in your screenshot can be ignored, this VMT info is more for me than for the user

**Comment by xiawiner** [comment]
Today I downloaded the latest github file and compiled it, but the error still exists. No matter if I use other injectors, it still has the same error. After adding AI to the practice mode, it will start to stutter or even crash. I opened the error log file of dota2 and it was described as accessing an inaccessible memory address. Has anyone encountered this situation?

**Comment by xiawiner** [comment]
[WARNING] Source2EngineToClient0: 0x7ffc93f21520 | VM count mismatch! Current: 180, Expected: 177
[WARNING] VFileSystem: 0x7ffc93a71660 | VM count mismatch! Current: 137, Expected: 136
[WARNING] SchemaSystem: 0x7ffca30fc730 | VM count mismatch! Current: 42, Expected: 38
![b](https://github.com/ExistedGit/Dota2Cheat/assets/20815754/09f1e30e-61b6-42e4-9f66-e9128bc4c7b0)

**Comment by xiaplayer** [comment]
I have the same error,my friend, I hope the author can release the latest version

**Comment by xiawiner** [comment]
> I have the same error,my friend, I hope the author can release the latest version

yeah,I downloaded the latest version, but still got the same error. The previous compilation was directly available.
